android/src/app/organicmaps/settings/SettingsPrefsFragment.java

```diff
@@ -573,12 +542,32 @@ public boolean onPreferenceChange(Preference preference, Object newValue)
     });
   }
 
+  private void disableOrEnable3DBuildingsForPowerMode(Integer powerManagementValue)
```

Comment:
Adding a comment that the `null` case is valid if the preference is not initialized can be helpful.

Reply:
Added comment